.github/workflows/trigger-webui.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow checks if the PR affects Anaconda (changes to pyanaconda folder),
+# polls the packit COPR until it has the current PR version
+# available, and then test-triggers an "anaconda PR" scenario.
+#
+# This workflow is inspired from:
+# https://github.com/cockpit-project/cockpit/blob/main/.github/workflows/trigger-anaconda.yml
+
+name: Anaconda Web UI
+on:
+  pull_request_target:
+    # All file changes that might affect the Web UI
+    paths:
+      - 'pyanaconda/**'
+      - '!pyanaconda/gui/**'
+      - '!pyanaconda/tui/**'
+      - 'anaconda.py'
+      - 'data/anaconda.conf'
+      - 'data/conf.d/**/'
+      - 'data/profile.d/**'
+jobs:
+  trigger:
+    runs-on: ubuntu-22.04
+    # the default workflow token cannot read our org membership, for deciding who is allowed to trigger tests
+    environment: gh-cockpituous
+    container: registry.fedoraproject.org/fedora:rawhide
+    # this polls for a COPR build, which can take long
+    timeout-minutes: 120
+
+    steps:
+      - name: Install dependencies
+        run: |
+          dnf install -y git-core dnf-plugins-core || {
+            sleep 60
+            dnf install -y git-core dnf-plugins-core
+          }
```

Comment:
Why do this?

Reply:
This was more or less copied from the original workflow but here is the explanation from martin who wrote the original:

`this is a "transient network failure" retry; that's simply to increase the chance that the second attempt will work. You can certainly drop it at least initially, but at some point you want it back; fetching package indexes is rather error prone unfortunately`

Reply:
Oh. If it's copied from some proven solution, then that's it. Still, this never fails in container builds, so my confusion remains... but reality sometimes defies logic :-/